scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
+              help='Portal to add the analyses to')
+@click.option('--analyses', required=True, type=str,
+              help='Comma seperated list of analyses to add')
+def add_analysis(portal, analyses):
+    ids = map(int, analyses.split(','))
+    Portal(portal).add_analyses(ids)
```

Comment:
For these command line interfaces, you probably want to catch the possible exceptions and raise one of Click's [exceptions](http://click.pocoo.org/4/exceptions/#which-exceptions-exist). That way the traceback is avoided and a message is just printed out :smiley: